charts/opensearch/templates/_helpers.tpl

```diff
@@ -52,14 +52,10 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 
 {{- define "opensearch.majorVersion" -}}
-{{- if .Values.majorVersion -}}
-{{ .Values.majorVersion }}
-{{- else -}}
-{{- $version := int (index (.Values.imageTag | splitList ".") 0) -}}
-  {{- if and (contains "opensearchproject/opensearch" .Values.image) (not (eq $version 0)) -}}
-{{ $version }}
-  {{- else -}}
-7
-  {{- end -}}
-{{- end -}}
-{{- end -}}
+{{- if .Values.majorVersion }}
+  {{- .Values.majorVersion }}
+{{- else }}
+  {{- $version := semver (coalesce .Values.imageTag .Chart.AppVersion "7") }}
```

Comment:
Why we have made it `7`, shouldn't it be `1`?

Reply:
The only reason that it is `7` is that I copied the fallback value that already existed (take a look at the diff to see this).

Should it be changed to `1`?

Reply:
Might be it was missed, but I feel we should have `1` as `7` is not yet released.  I don't think people will use this chart to install `elasticsearch` cluster.

@DandyDeveloper your views on this?

Reply:
CC : @peterzhuamazon

Reply:
No need to include ODFE as it is in maintenance only mode.